releasenotes/notes/fix-itr-in-evp-mode-e66fa0ef8a6ffe72.yaml

```
@@ -0,0 +1,4 @@
+fixes:
+  - |
+    CI Visibility: fixes an issue where the Intelligent Test Runner would not work
+    when in EVP proxy mode due to missing X-Datadog-NeedsAppKey header.
```

It might be a better idea to type ``X-Datadog-NeedsAppKey`` instead so `build_docs` job passes.
```suggestion
    when in EVP proxy mode due to missing ``X-Datadog-NeedsAppKey`` header.
```